index.js

```diff
@@ -29,20 +29,15 @@ function getJSON(obj) {
 		} else {
 			obj.headers = {'User-Agent': FAKE_USER_AGENT};
 		}
-		request({
-			url: obj.url,
-			headers: obj.headers,
-			rejectUnauthorized: false
-		}, function (err, req, body) {
-			if (err) {
-				return reject(err);
-			}
+		got(obj.url, {
+			headers: obj.headers
+		}).then(request => {
```

Comment:
`request` would be named `response` more appropriately. The variables `request` and `response` are common in node and developers expect to be able to do certain things with these objects. This is a bit confusing.